_posts/2016-09-23-custom-flows-parsing-xml-part-2.md

```diff
@@ -51,7 +51,7 @@ new GraphStageLogic(shape) with InHandler with OutHandler {
        val coalesced = buffer.toString()
        isBuffering = false
        buffer.clear()
-       emit(out, Characters(coalesced), () => emit(out, other))
+       emitMultiple(out, Iterable(Characters(coalesced), other))
```

Comment:
Yeah that's how I've done it in the past, the emitMultiple docs use Iterable (see http://doc.akka.io/docs/akka/2.4.10/scala/stream/stream-customize.html and duplicate example).  Happy to switch it to List() or :: Nil though.